src/odemis/gui/util/img.py

```diff
@@ -848,16 +848,21 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
         logging.info("Trying to compute legend tick without range")
         return None
 
-    min_val, max_val = value_range[0], value_range[-1]
+    reverse = value_range[0] > value_range[-1]
+    if reverse:
+        min_val, max_val = value_range[-1], value_range[0]
+    else:
+        min_val, max_val = value_range[0], value_range[-1]
```

Comment:
I think range is useally just a tuple of two isn't it?
Can this not be shortened into:
> min_val, max_val = min(value_range), max (value_range)

or maybe better:
> min_val = min(value_range)
> max_val = max (value_range)

To me this is clearer and uses less code but maybe value_range is more complicated than I think/see

Reply:
Actually, that was the main previous change by Victoria, now "range_value" can be a list. I agree with you that we could avoid using the if to make the code a little bit easier to read. Maybe something like this:
```python
min_val = min(value_range[0], value_range[-1])
max_val = max(value_range[0], value_range[-1])
```

Reply:
yes you are right, as long as I don't use reverse again, I can immediately do it with min and max